packages/components/src/__future__/Tag/Tag/_docs/Tag-migration-guide.stories.tsx

```diff
@@ -23,106 +22,31 @@ const meta = {
 } satisfies Meta<typeof Tag>
 
 export default meta
+type Story = StoryObj<typeof meta>
 
-/** * This is a stand-in component for the legacy Tag's bake in LiveIcon - we should consider adding this as an actual component or replacing it */
-const LiveIconComponent = (): JSX.Element => (
-  <span className={styles.liveIcon}>
-    <LiveIcon
-      role="presentation"
-      classNameOverride={styles.liveIcon_base}
-      width="16"
-      height="16"
-      viewBox="0 0 16 16"
-      fill="none"
-    />
-    <LiveIcon
-      role="presentation"
-      classNameOverride={styles.liveIcon_1}
-      width="16"
-      height="16"
-      viewBox="0 0 16 16"
-      fill="none"
-    />
-    <LiveIcon
-      role="presentation"
-      classNameOverride={styles.liveIcon_2}
-      width="16"
-      height="16"
-      viewBox="0 0 16 16"
-      fill="none"
-    />
-    <LiveIcon
-      role="presentation"
-      classNameOverride={styles.liveIcon_3}
-      width="16"
-      height="16"
-      viewBox="0 0 16 16"
-      fill="none"
-    />
-  </span>
-)
-
-export const LiveIconComponentStory: StoryObj = {
-  render: () => <LiveIconComponent />,
+export const LiveIconComponentStory: Story = {
+  render: () => <LiveIcon />,
   parameters: {
     docs: {
       source: {
         type: 'dynamic',
         code: `
-  // component with styled with CSS modules
-  const LiveIconComponent = (): JSX.Element => (
-    <span className={styles.liveIcon}>
-      <LiveIcon
-        role="presentation"
-        classNameOverride={styles.liveIcon_base}
-        width="16"
-        height="16"
-        viewBox="0 0 16 16"
-        fill="none"
-      />
-      <LiveIcon
-        role="presentation"
-        classNameOverride={styles.liveIcon_1}
-        width="16"
-        height="16"
-        viewBox="0 0 16 16"
-        fill="none"
-      />
-      <LiveIcon
-        role="presentation"
-        classNameOverride={styles.liveIcon_2}
-        width="16"
-        height="16"
-        viewBox="0 0 16 16"
-        fill="none"
-      />
-      <LiveIcon
-        role="presentation"
-        classNameOverride={styles.liveIcon_3}
-        width="16"
-        height="16"
-        viewBox="0 0 16 16"
-        fill="none"
-      />
-    </span>
-  )
+          import { LiveIcon } from '@kaizen/components/Tag/subcomponents'
```

Comment:
Haha, not quite! We don't actually want consumers to be using this, so I'd reinstate the original code snippet where consumers need to create their own custom animated live icon.
Moving forward, either we make a proper animated live icon, or we tell people that animated ones are no longer recommended (TBC with design).

Reply:
Oh that's my bad, I thought we were exporting from the old tag as a final part of the migration